OpenRA.Game/Map/MapCache.cs

```diff
@@ -48,12 +53,22 @@ public MapCache(ModData modData)
 			sheetBuilder = new SheetBuilder(SheetType.BGRA);
 		}
 
+		public void UpdateMaps()
+		{
+			foreach (var tracker in mapDirectoryTrackers)
+			{
+				tracker.UpdateMaps();
+			}
```

Comment:
For things on a single line, our style says to leave off the braces.

Reply:
ok